app/res/layout/app_manager.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" >
+
+    <ImageView
+        android:id="@+id/screen_bulk_image1"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginRight="20dp"
+        android:maxHeight="135dp"
+        android:maxWidth="663dp"
+        android:paddingBottom="0dp"
+        android:paddingLeft="20dp"
+        android:paddingTop="20dp"
+        android:scaleType="fitStart"
+        android:src="@drawable/ccbanner" />
+    
+    <TextView
+        android:id="@+id/manager_instructions"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/manager_instructions" 
+        android:layout_below="@id/screen_bulk_image1"
+        android:paddingTop="10sp"
+        android:paddingBottom="10sp"
+        android:textSize="23sp" 
+        android:textColor="#616161"
```

Comment:
Add this textSize and textColor to dimens.xml and colors.xml - or, preferably, use one of the standard colors and text sizes in those files.